contrib/database/sql/option.go

```diff
@@ -88,6 +89,16 @@ func WithChildSpansOnly() Option {
 	}
 }
 
+// WithCustomTag will attach the value to the span tagged by the key.
```

Comment:
```suggestion
// WithCustomTag will attach the key:value tag to all spans.
```
Is this correct?

For some reason the current explanation is a bit confusing.